src/components/specific/models/models-manager/models-table/model-name-cell/ModelNameCell.vue

```diff
@@ -40,16 +40,16 @@
 
       <div v-else class="model-name-cell__content">
         <template v-if="model.type === MODEL_TYPE.IFC">
-          <img src="/static/ifc-logo.svg" />
+          <BIMDataIcon :name="MODEL_ICON.IFC" size="m" />
         </template>
         <template v-if="model.type === MODEL_TYPE.DWG">
-          <BIMDataIcon name="fileDwgPolychrome" size="m" />
+          <BIMDataIcon :name="MODEL_ICON.DWG" size="m" />
         </template>
         <template v-if="model.type === MODEL_TYPE.DXF">
-          <img src="/static/dxf-file.svg" />
+          <BIMDataIcon :name="MODEL_ICON.DXF" size="m" />
         </template>
         <template v-if="model.type === MODEL_TYPE.PDF">
-          <BIMDataIcon name="filePdfPolychrome" size="m" />
+          <BIMDataIcon :name="MODEL_ICON.PDF" size="m" />
         </template>
         <template v-if="model.type === MODEL_TYPE.META_BUILDING">
           <BIMDataIcon name="building" size="s" />
```

Comment:
I have a proposal to improve these lines:
```html
<template v-if="model.type === MODEL_TYPE.META_BUILDING">
  <BIMDataIcon name="building" size="s" />
</template>
<template v-else>
  <BIMDataIcon :name="MODEL_ICON[model.type]" size="m" />
</template>
```

This would be shorter and more maintainable. ;-)